services/ops/AgentWatcher/README.md

```diff
@@ -0,0 +1,32 @@
+## Agent Watcher
+
+The Agent Watcher is used to monitor agents running on a VOLTTRON instance. Specifically it monitors whether a set of 
+VIP identities (peers) are connected to the instance. If any of the peers in the set are not present then an alert will 
+be sent.
+
+### Configuration
+
+The agent has two configuration values:
+
+* watchlist: a list of VIP identities to watch on the platform instance
+* check-period: interval in seconds between the agent watcher checking the platform peerlist and publishing alerts
+
+
+    {
+        "watchlist": [
+            "platform.driver",
+            "platform.actuator"
+        ],
+        "check-period": 10
+    }
+
```

Comment:
this block does not render as code on github: (https://github.com/jklarson/volttron/tree/docs/ops_readme_md/services/ops/AgentWatcher)

Reply:
resolved